.testing/Makefile

```diff
@@ -218,13 +218,20 @@ else
 MPIRUN_CMD=$(MPIRUN) $(if $(1),-x $(1),)
 endif
 
+# Rule to build <tc>/ocean.stats.<tag>
+# $(1) - <tag>
+# $(2) - <executable-label>
+# $(3) - report to codecov
+# $(4) - Parameter line for MOM_override
+# $(5) - Environment for MPIRUN
+# $(6) - Number of PEs
```

Comment:
I had these defined up in line 205, did you also want to remove the other one?

Reply:
Ha! I didn't see those comments. It doesn't seem right to have an `ifeq` with it's own comments in between the description and function. I'll delete mine and move yours down?

Reply:
Yeah I think that works.  You can just delete mine if it's easier (or merge them into a grand unified comment)

Reply:
I rearranged and combined. PR has been updated.